src/TSHelper.ts

```diff
@@ -33,6 +33,21 @@ const defaultArrayCallMethodNames = new Set<string>([
     "flatMap",
 ]);
 
+const builtinErrorTypeNames = new Set([
```

Comment:
Could you mark this with a `// TODO [2019-09-27/Perry]: Refactor lualib detection to consistent map` :)